Dockerfile

```diff
@@ -1,8 +1,9 @@
 # Stage: base image
-FROM node:20.10-bullseye-slim as base
+FROM node:20.11-bookworm-slim as base
 
-ARG BUILD_NUMBER=1_0_0
-ARG GIT_REF=not-available
+ARG BUILD_NUMBER
+ARG GIT_REF
+ARG GIT_BRANCH
```

Comment:
CircleCI passes these into the docker build:

<img width="775" alt="Screenshot at Mar 28 13-07-47" src="https://github.com/ministryofjustice/hmpps-education-and-work-plan-ui/assets/125488090/92407312-b644-473e-8335-a286ff8e3b38">